Add README

tverghis opened the pull request:
I think I've covered all the common commands. I'm not 100% sure about the `!9kmmrbot emotes add/del` commands, so let me know if they need revision.

Closes #18.

tverghis committed:
Add README

Hambergo commented:
for adding notable players/ global notable players, the nickname is required and not optional. also you can add that just the friend id (from dota/dotabuff/opendota/stratz also works) and everything is converted to it
toggleemotes note: they only need to whisper if they want to add an emote to the bot
delay: maybe specify that it's not making the bot response with a delay, but make the response for X seconds in the past instead
id note: it should be case insensitive

the emotes add/del gets a emote or emote combo, and a hero, if the hero is valid, and the bot has all the emotes in the list, it adds the emotes as a possible. an example for emote combo is capitalist's sk/earthshaker emotes 
same for deleting, (so can't delete emotes if the bot doesn't have them)

the pastebin environment variables aren't really necessary now, because they're only used in the commands of recent/shared matches that are experimental/won't be released to the public as of now
NODE_ENV=production is only needed if you want the bot to write in the channels
the rest of the environment variables are necessary

tverghis commented:
@Hambergo ready for review.

1. Opened #22 
2. Removed that they need to whisper you; that is specific to the current instance of the bot. It makes no sense if someone else spins up their own instance. Updated the wording to reflect how it should be done.
